bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.6.3",
+  "version": "1.0.0",
```

Comment:
Feeling is that Syphon has depended on jQuery for so long now that changing it for v1 would potentially break code and assumptions.

I do agree that it would be nicer to remove the dependencies going forward. I do see a lot of code transitioning away from jQuery over time.

Perhaps we resolve to remove jQuery for v2?